modules/auxiliary/scanner/http/surgenews_user_creds.rb

```diff
@@ -0,0 +1,193 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Auxiliary::Report
+  include Msf::Auxiliary::Scanner
```

Comment:
The order in which the mixins are loaded is causing the host header issue I was getting. Moving `include Msf::Auxiliary::Scanner` to the bottom (to be included last), the module successfully works.